.github/workflows/deploy.yml

```
@@ -22,27 +22,33 @@ jobs:
           java-version: '11' # Robolectric requires v9, but we choose LTS: https://adoptopenjdk.net/
       - name: Setup Android SDK
         uses: android-actions/setup-android@v2
-      - name: Prepare app for building 
+      - name: Prepare app for building
         run: echo "${{ secrets.GOOGLE_SERVICES_BASE64 }}" | base64 -d > app/google-services.json
         env:
           GOOGLE_SERVICES_BASE64: "${{ secrets.GOOGLE_SERVICES_BASE64 }}"
-      - name: Setup Fastlane 
+      - name: Create Local Properties File
+        run: touch local.properties
+      - name: Add CIO creds secret in local properties
+        run: |
+          echo "siteId=${{ secrets.CIO_SITE_ID }}" >> local.properties
+          echo "apiKey=${{ secrets.CIO_API_KEY }}" >> local.properties
```

```suggestion
      - name: Create Local Properties File
        run: |
          touch local.properties
          echo "siteId=${{ secrets.CIO_SITE_ID }}" >> local.properties
          echo "apiKey=${{ secrets.CIO_API_KEY }}" >> local.properties
```

Do we see value in combining these 2 steps to keep this file smaller in size? I believe it makes it a little more readable since all 3 of these run steps are working to accomplish the 1 task of creating a local.properties file.